custom/bin/ARE/data/webservice/angular/envControl/component/help/faq/html/faq3de.html

```diff
@@ -1,9 +1,13 @@
-<p>Es kann nicht verallgemeinert werden, welche zusätzliche Hardware erworben werden muss. Dies hängt davon
-    ab, welche Geräte mit AsTeRICS Ergo gesteuert werden sollen.
-    Um herauszufinden, welche zusätzliche Hardware gekauft werden muss, verwenden Sie den
-    <a href="" ng-click="$ctrl.toState('asterics.envControl.STATE_HELP_DEVICES')">Hardware-Assistenten</a>.
-    Wählen sie alle Geräte, die gesteuert werden sollen, aus. Die
-    Hardware, die Sie zur Steuerung benötigen, wird darunter angezeigt.
+<p>Es kann nicht verallgemeinert werden, welche zusätzliche Hardware erworben werden muss. Dies hängt davon ab, welche Geräte mit AsTeRICS Ergo gesteuert werden sollen.
+    Verwenden Sie den
+    <a href="" ng-click="$ctrl.toState('asterics.envControl.STATE_HELP_DEVICES')">Bestell-Assistenten</a> um herauszufinden, welche zusätzliche Hardware gekauft werden muss.
+    Wählen sie alle Geräte, die gesteuert werden sollen, aus. Die Hardware, die Sie zur Steuerung benötigen, wird darunter angezeigt.
+1. Möchten Sie ein Gerät steuern, das normalerweise per Fernbedienung bedient wird, benötigen Sie für die Steuerung einen IrTrans. Dieser ermöglicht es Fernbedienungsbefehle zu ersetzen. Es muss immer nur ein IrTrans erworben werden, da dieser mehrere Fernbedienungsgeräte steuern kann. Außer dem IrTrans ist für die Steuerung dieser Geräte keine weitere zusätzliche Hardware notwendig. 
```

Comment:
This enumeration will not be very nice, its better to use a HTML-element like <ol>. I'll show you how - maybe on monday.

Reply:
see https://github.com/asterics/AsTeRICS-Ergo/pull/62/files how to use `<ol>`. it is a "ordered list" and HTML does generate the numbers automatically.